Userland/Libraries/LibJS/Tests/test-common.js

```diff
@@ -306,14 +306,30 @@ class ExpectationError extends Error {
                     threw = false;
                 } catch (e) {
                     if (typeof value === "string") {
-                        this.__expect(e.message.includes(value));
+                        this.__expect(
+                            e.message.includes(value),
+                            `Expected ${this.target.toString()} to throw and message to include ${value} but message ${
+                                e.message
+                            } did not contain it`
```

Comment:
Should probably quote `value` and `e.message` here to make it clear these are strings